### PR TITLE
Add six tests for AWS triggers

### PR DESCRIPTION
**Pull request by slycyberguy**
Unit tests added for amazon/aws/triggers: rds, lambda_function, glue_crawler, emr, batch, athena

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/contributing-docs/05_pull_requests.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (eladkal) [inline .github/workflows/ci.yml:1]: Why are you removing this file?

Comment (slycyberguy) [reply .github/workflows/ci.yml:1]: On my local repo, I added CodeCov to the file and I accidentally committed it. So I removed the update.

Comment (potiuk) [reply .github/workflows/ci.yml:1]: So you should bring it back - otherwise no tests are run at all and if we merge it, it will break our CI.

Comment (slycyberguy) [reply .github/workflows/ci.yml:1]: That makes sense. I have now added the original file to the PR. Sorry, I'm new to GitHub. At first, I thought the one in your repo wouldn't be affected. Let me know if there are other actions I need to take.

Comment (potiuk) [reply .github/workflows/ci.yml:1]: Still not there

Comment (slycyberguy) [reply .github/workflows/ci.yml:1]: I added it to PR #38809. That's where it was deleted. Do I need to add it to this PR as well? Originally, all the unit tests and the ci.yml deletion were on that PR (#38809), but I wanted to separate it into two PRs. So I force pushed the removal of unit tests for aws/triggers from that PR (#38809). Then I created a new branch 'alt' and submitted a new PR (this one #38810) with the trigger tests, but for some reason the commits from my 'main' branch is showing up here.

Comment (potiuk) [reply .github/workflows/ci.yml:1]: each PR is separate. You should not remove the .ci.yml in your main. Look at the list of changed files in this PR.

Comment (slycyberguy) [reply .github/workflows/ci.yml:1]: Yes, I see that this PR says the file is deleted. So what steps should I take? I already pushed the addition of the file into the other PR.

Comment (potiuk) [reply .github/workflows/ci.yml:1]: Restore the file, the usual way.

We generally don't teach git basics, the expectation is that you learn it from some dedicated tutorials.

I am not sure what to advice you as I am not sure what clients and tooling you use but I'd personally use some combination of `git reset HEAD^`, `git add -p`, `git rebase`, `git commit -m`, `git push -f`. But it's up to you and you will have to learn all the basic things ( but also  git rebase) in order to contribute. 

BTW. Were the tests generated by an AI maybe? They look like generated.

Comment (slycyberguy) [reply .github/workflows/ci.yml:1]: Thanks for the advice. I'll take care of it.

I'm new to unit testing, so I started off with developing unit tests for simpler services. I used AI on a couple to help me get started. Then I kept everything else consistent as I coded the rest.

**Pull request closed without merging**